src/libnml/inifile/inifile.cc

```diff
@@ -315,9 +315,8 @@ IniFile::Find(const char *_tag, const char *_section, int _num, int *lineno)
            continue; // get next line to extend
         } else {
             if (extend_ct) {
-               strncpy(elinenext,line,newLinePos);
+                snprintf(elinenext, newLinePos, "%.*s", newLinePos, line);
```

Comment:
Using newLinePos twice here seem redundant, as both uses will limit the string length.  Why did you both use snprintf() to limit the length, and %.*s to do the same?

Reply:
dropped during rebase